docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+* A Diff DID message is NOT allowed to add, remove or update any keys used to sign the Diff DID messages. This must be done via an Integration DID message.
+* A Diff DID message must be published to an IOTA Tangle on an index that is generated by the hash, generated by the `BLAKE2b-256` hashing algorithm, of the `previous_message_id` of the latest Integration DID message and encoded in `hex`. 
+* Diff DID Messages have at least the following attributes:
+    * `id` (REQUIRED): This field helps link the update to a DID. The value of `id` MUST be a string that references the DID that this update applies to. 
+    * `previous_message_id` (REQUIRED): This field provides an immutable link to the previous DID document that is updated and is used for basic ordering of the DID messages, creating a chain. The value of `previous_message_id` MUST be a string that contains an IOTA MessageId from the previous DID message it updates, which references either a Diff or Int Chain message. Read the [Previous Message Id](#previous-message-id) section for more information.
+    * `diff` (REQUIRED): A Differentiation object containing all the changes compared to the DID Document it references in the `previous_message_id` field. The value of `diff` MUST be an escaped JSON string following the [Autonomy of the Diff object](#autonomy-of-the-diff-object) definition.
+    * `proof` (REQUIRED): This field provides a cryptographic proof on the message that proves ownership over the DID Document. The value of the `proof` object MUST contain an object as defined by [Autonomy of the Proof object](#autonomy-of-the-proof-object).
+
+Example of a Diff DID message:
+```json
+{
+  "did": "did:iota:7EhyBxAhFXojqzrKt8Zq7QBvxNZWJJ4xj1mm2QgLmcKj",
+  "diff": "{\"authentication\":[{\"index\":0,\"item\":{\"Embed\":{\"id\":\"did:iota:7EhyBxAhFXojqzrKt8Zq7QBvxNZWJJ4xj1mm2QgLmcKj#key-3\",\"key_data\":{\"PublicKeyBase58\":\"TJqJAnV387wTUfzq8BVE7iJ9LYs7xJYM4SEF86LkB8E\"}}}}],\"properties\":[{\"c:k\":\"updated\",\"c:v\":\"2021-04-15T10:31:21Z\"}]}",
+  "previous_message_id": "9cd2e34c049099246d247ffcf19ba0d54063add9cb7787662b5d51a2a36a8a3b",
+  "proof": {
+    "type": "JcsEd25519Signature2020",
+    "verificationMethod": "#key-2",
+    "signatureValue": "G4otnurZMiXwKYpFxZ5Nk99R95FZQ2YSYjztDosaXbyDz8qFrzH8FHbLzFp8doUEPuyHN6CPm58QbAKUqrvq8PV"
+  }
+}
+```
+
+### Autonomy of the Proof object
+
+Following the proof format in the [Verifiable Credential standard](https://www.w3.org/TR/vc-data-model/#proofs-signatures), at least one proof mechanism, and the details necessary to evaluate that, MUST be expressed for a DID Document uploaded to the Tangle. 
+
+The proof object is an embedded proof that contains all information to be verifiable. It contains one or more cryptographic proofs that can be used to detect tampering and verify the authorship of a DID creation or update. It mostly follows LD-Proofs standard.
```

Comment:
Could you elaborate on the "mostly" part of "mostly follows LD-Proofs standard"? Are there material differences that an implementer would need to know about?

Reply:
@m-renaud see above https://github.com/iotaledger/identity.rs/pull/207/files#r625389350

we currently only support one signature suite and are missing some of the other features like `proofChains`